genesis/process/testdata/smartcontracts.json

```diff
@@ -10,8 +10,8 @@
     "owner": "0102030405060708090001020304050607080900010203040506070809000102",
     "filename": "testdata/delegation.wasm",
     "vm-type": "0500",
-    "init-parameters": "0BB8@%auction_sc_address%@0A61D0",
+    "init-parameters": "%auction_sc_address%@03E8@03E8@03E8",
```

Comment:
we need a tag somewhere and release version of contract before mainnet. something official.

Reply:
Totally agree, with hash verification. Let's plan this through.